if.go

```diff
@@ -46,29 +49,14 @@ func New(config Config) (ifce *Interface, err error) {
 	if zeroConfig == config {
 		config = defaultConfig()
 	}
-	return newDev(config)
-}
-
-// NewTAP creates a new TAP interface whose name is ifName. If ifName is empty, a
-// default name (tap0, tap1, ... ) will be assigned. ifName should not exceed
-// 16 bytes. TAP interfaces are not supported on darwin.
-// ifName cannot be specified on windows, you will need ifce.Name() to use some cmds.
-//
-// Note: this function is deprecated and will be removed from the library.
-// Please use New() instead.
-func NewTAP(ifName string) (ifce *Interface, err error) {
```

Comment:
Ah I see! I remember now why they were all using separate functions. Thanks for pointing it out. Having a `if_linux.go` sounds good to me; Linux was the only platform supported at that time any way.